### PR TITLE
RFC-0311 Technical Editing

### PR DESCRIPTION
**Pull request by anselld**
## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [x] Documentation

## Checklist:

* [x] I'm merging against the `development` branch
* [ ] I ran `cargo-fmt --all` before pushing
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by anselld** [comment]
Hi @hansieodendaal, I'm unsure why RFC-0150 keeps on appearing as a conflicting file. Do I just delete it?

File: RFC/src/RFC-0311_AssetTemplates.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # RFC-0311/AssetTemplates
 
-## Digital Asset templates
+## Digital Asset Templates
 
 ![status: draft](https://github.com/tari-project/tari/raw/master/RFC/src/theme/images/status-draft.svg)
```
Comment (hansieodendaal) [inline RFC/src/RFC-0311_AssetTemplates.md]:
```suggestion
![status: draft](theme/images/status-draft.svg)
```

**Comment by anselld** [comment]
Hi @hansieodendaal , thanks for the suggestion - accepted.

**Comment by hansieodendaal** [comment]
@anselld thanks, all good, you can combine your commits.

**Comment by anselld** [comment]
Hi @hansieodendaal , I've pulled on the branch, combined the commits in the log, and pushed to Github, but I still see eight commits in the top line. Am I doing something wrong?

File: RFC/src/RFC-0311_AssetTemplates.md [hunk 2]
```diff
@@ -22,148 +22,148 @@ following conditions are met:
...
-| **template-specific data**         | Object        | Template-specific metadata can be defined in this section                                                                              |
-| **Signatures**                     |               |                                                                                                                                        |
-| metadata_hash                      | `u256`        | A hash of the previous three sections' data, in canonical format (`m`)                                                                 |
-| creator_sig                        | `u256`        | A digital signature of the message `H(R ‖ P ‖ RAID_ID ‖ m)` using the asset creator’s private key corresponding to the `issuer` PKH |
-| commitment_sig                     | `u256`        | A signature proving the issuer is able to spend the commitment to the asset fee                                                        |
-
-#### Committee parameters
-
-If `committee_mode` is `CREATOR_NOMINATION` the `committee_parameters` object is
+| Name                               | Type          | Description                                                  |
```
Comment (CjS77) [inline RFC/src/RFC-0311_AssetTemplates.md]: All the formatting for this table is broken. The Colons should be nicely aligned as they were before.

Comment (hansieodendaal) [reply RFC/src/RFC-0311_AssetTemplates.md]: Hi @anselld, just copy the table only, paste it in [this tool](http://markdowntable.com/), format and copy the result. It nicely formats Markdown tables for easier reading when viewed with a pure text editor (although nothing is technically wrong with the table).

I see the table here
_If `committee_mode` is `PUBLIC_NOMINATION`, the `committee_parameters` object is:_
also has some misaligned colons at the end. You could also fix that.

Thanks!

Comment (anselld) [reply RFC/src/RFC-0311_AssetTemplates.md]: Hi @hansieodendaal , I don't see the problem... please explain.
Thanks and regards
Denise

Comment (anselld) [reply RFC/src/RFC-0311_AssetTemplates.md]: Hi @CjS77 , tables have been fixed - thanks for spotting that.

**Pull request merged**